CENSUS_METADATA.md

```diff
@@ -0,0 +1,778 @@
+| `Release_Date` | `String!` | `Census_Release.Release_Date` | |
+
+## Statistical_Unit
+
+The data is sourced from `Statistical_Unit.csv`. The `Id` field in `Statistical_Unit.csv` is ignored.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `Statistical_Unit` | `String!` | `Statistical_Unit.Statistical_Unit` | |
+| `Statistical_Unit_Description` | `String!` | `Statistical_Unit.Statistical_Unit_Description` | `Statistical_Unit.Statistical_Unit_Description_Welsh` |
+
+
+# Sample queries
+
+This section contains some examples of GraphQL queries that can be used to query metadata for the
+dataset name `Teaching-Dataset` from `cantabular-metadata` and `cantabular-api-ext`. Sample responses
+have also been provided.
+
+## `cantabular-metadata` queries
+
+### Get all tables
+
+This query gets the `name` value for every table in the metadata.
+
+#### Query
+
+```
+{
+  service {
+    tables {
+      name
+    }
+  }
+}
+```
+
+#### Response
+
+```
+{
+  "data": {
+    "service": {
+      "tables": [
+        {
+          "name": "LC1117EW"
+        },
+        {
+          "name": "LC2101EW"
+        },
+        {
+          "name": "LC2107EW"
+        },
+        {
+          "name": "LC6107EW"
+        },
+        {
+          "name": "LC6112EW"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Get metadata for one table
+
+This query gets additional metadata for a single table named **LC1117EW**.
+
+#### Query
+
+```
+{
+  service {
+    tables(names: ["LC1117EW"]) {
+      name
+      label
+      description
+      meta {
+        Alternate_Geographic_Variables
+        Statistical_Unit {
+          Statistical_Unit_Description
+        }
+      }
+      datasetName
+      vars
+    }
+  }
+}
+```
+
+#### Response
+
+```
+{
+  "data": {
+    "service": {
+      "tables": [
+        {
+          "datasetName": "Teaching-Dataset",
+          "description": "This dataset provides 2011 Census estimates that classify usual residents by sex, and by age (ages 0 to 15 grouped together, 10 year age groups up to 74 then 75 years and over grouped together). The estimates are as at census day, 27 March 2011.",
+          "label": "Sex by age",
+          "meta": {
+            "Alternate_Geographic_Variables": [
+              "Country"
+            ],
+            "Statistical_Unit": {
+              "Statistical_Unit_Description": "People living in England and Wales"
+            }
+          },
+          "name": "LC1117EW",
+          "vars": [
+            "Region",
+            "Sex",
+            "Age"
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+### Get metadata for dataset and variables
+
+This query gets some metadata for a specific dataset named **Teaching-Dataset** along
+with metadata for the **Region** and **Sex** variables.
+
+#### Query
+
+```
+{
+  dataset(name: "Teaching-Dataset") {
+    name
+    description
+    meta {
+      Lowest_Geog_Variable
+    }
+    vars(names: ["Region" "Sex"]){
```

Comment:
```suggestion
    vars(names: ["Region", "Sex"]){
```

Reply:
The syntax I have is valid. Commas are "Ignored Tokens":
https://spec.graphql.org/October2021/#sec-Language.Source-Text.Ignored-Tokens

However, I'll add for readability.